swot/README.md

```diff
@@ -0,0 +1,5 @@
+# IOMUST: internet of musical things ontology
```

Comment:
Looks like this be about SemanticWoT.  I'm assuming this was a copy & paste from previous PR.

Reply:
It's totally a copy paste error. Sorry about that, I'll fix the problem.
Thank you a lot.